taskcluster/fenix_taskgraph/target_tasks.py

```diff
@@ -30,10 +30,22 @@ def target_tasks_nightly(full_task_graph, parameters, graph_config):
     """Select the set of tasks required for a nightly build."""
 
     def filter(task, parameters):
-        return task.attributes.get("nightly", False)
+        return (
+            task.attributes.get("nightly", False) and
+            not _filter_fennec_nightly(task, parameters)
```

Comment:
Note: Fennec nightly is now out of the regular nightly graph because we don't want to mix up builds on the TC index.